src/abstractdataframe/selection.jl

```diff
@@ -1665,6 +1665,22 @@ julia> combine(gd, :, AsTable(Not(:a)) => sum, renamecols=false)
    7 │     4      1      4      5
    8 │     4      1      8      9
 ```
+
+# aggregation over array-like elements
```

Comment:
It is not aggregation specific. @nalimilan - what would be the best comment here?
Maybe:

```
# protecting vectors returned from a function from being expanded into multiple rows
```
(but it seems long, but maybe it is OK)